src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -425,6 +431,43 @@ public function denormalize(mixed $data, string $type, string $format = null, ar
             }
         }
 
+        foreach ($nestedAttributes as $property => $attribute) {
+            $value = $this->extractNestedValue($attribute, $normalizedData);
+            $attributeContext = $this->getAttributeDenormalizationContext($resolvedClass, $property, $context);
+            if (null !== $value) {
```

Comment:
`if (null === $value)` then continue to unindent the remaining
also, no need to compute `$attributeContext` if `$value` is null, isn't it?